gum/arch-x86/gumx86writer.c

```diff
@@ -784,11 +800,22 @@ gum_x86_writer_put_call_address (GumX86Writer * self,
   }
   else
   {
+    gpointer label_jmp, label_addr;
+
     if (self->target_cpu != GUM_CPU_AMD64)
       return FALSE;
 
-    gum_x86_writer_put_mov_reg_u64 (self, GUM_REG_RAX, address);
-    gum_x86_writer_put_call_reg (self, GUM_REG_RAX);
+    label_jmp = self->code + 1;
+    label_addr = self->code + 2;
```

Comment:
These two can be folded into the declarations and they can be made gconstpointer, one on each line.